src/Compilers/CSharp/Test/Semantic/IOperation/IOperationTests_IAnonymousFunctionExpression.cs

```diff
@@ -21,7 +21,7 @@ class Program
 {
     static void Main(string[] args)
     {
-        Action x /*<bind>*/= () => F()/*</bind>*/;
+        /*<bind>*/Action x = () => F();/*</bind>*/
```

Comment:
before we moved node around to return something. for example, equal value returned declaration statement. now we always return what we got. we only return IOperation with declaration statement when we got declaration statement for GetOperation. 

there is no matching IOperation for equal value in this case or varaible decleration.

Reply:
Do we have a test that requests operation for a VariableDeclaration: `Action x = () => F()`?

Reply:
Why will the VariableDeclarationSyntax not return an IVariableDeclaration?

Reply:
I meant `VariableDeclaratorSyntax` not `VariableDeclarationSyntax`...

Reply:
in Operation tree, there is no matching IOperation concept for VariableDeclarationSyntax. if user call GetOperation with it, it will return null.

in IOperation, statement points declarator directly without middle decleration like syntax node.